src/generics.rst

```diff
@@ -300,12 +330,18 @@ of an :t:`associated trait type`.
 
 :dp:`fls_al4dhmqodvwc`
 A :t:`constant argument` may only appear as a single segment :t:`path
-expression`, optionally inside a :t:`block expression`, inside of a :t:`type` or
-:t:`array repeat expression`.
+expression`, optionally encapsulated in a :t:`block expression`, within an
+:t:`array repeat expression` or a :t:`type`.
 
 :dp:`fls_ukarc98ceesz`
 :t:`[Generic argument]s` are subject to :t:`generic conformance`.
 
+.. rubric:: Dynamic Semantics
+
+:dp:`fls_eZBsyyEOogEn`
+The :t:`evaluation` of a :t:`constant argument` evaluates its
+:t:`block expression` or :t:`literal expression`.
+
```

Comment:
This happens at compile time though, not run time, is that still dynamic?

Reply:
Oh, correct, I keep forgetting that constant XXX is **constant**. :D